tests/auth/providers/test_trusted_networks.py

```diff
@@ -5,10 +5,13 @@
 import pytest
 import voluptuous as vol
 
-from homeassistant import auth
+from homeassistant import auth, const
 from homeassistant.auth import auth_store
 from homeassistant.auth.providers import trusted_networks as tn_auth
 from homeassistant.data_entry_flow import RESULT_TYPE_ABORT, RESULT_TYPE_CREATE_ENTRY
+from homeassistant.setup import async_setup_component
+
+FORWARD_FOR_IS_WARNING = (const.MAJOR_VERSION, const.MINOR_VERSION) < (2021, 8)
```

Comment:
This is a temporary constant to automatically fail the tests when we bump the version. It's my intention to remove it directly in a follow-up PR such that this PR can be cherry-picked into the beta.